README.md

```diff
@@ -25,43 +25,49 @@
 
 **Meilisearch JavaScript** is the Meilisearch API client for JavaScript developers.
 
-**Meilisearch** is an open-source search engine. [Discover what Meilisearch is!](https://github.com/meilisearch/meilisearch)
+**Meilisearch** is an open-source search engine. [Learn more about Meilisearch.](https://github.com/meilisearch/meilisearch)
 
 ## Table of Contents <!-- omit in toc -->
 
 - [📖 Documentation](#-documentation)
 - [🔧 Installation](#-installation)
-- [🎬 Getting Started](#-getting-started)
-- [🤖 Compatibility with Meilisearch](#-compatibility-with-meilisearch)
-- [💡 Learn More](#-learn-more)
-- [⚙️ Development Workflow and Contributing](#️-development-workflow-and-contributing)
+- [🎬 Getting started](#-getting-started)
+- [🤖 Compatibility](#-compatibility-with-meilisearch)
+- [💡 Learn more](#-learn-more)
+- [⚙️ Contributing](#️-development-workflow-and-contributing)
 - [📜 API Resources](#-api-resources)
 
 ## 📖 Documentation
 
-See our [Documentation](https://docs.meilisearch.com/learn/tutorials/getting_started.html) or our [API References](https://docs.meilisearch.com/reference/api/).
+This readme contains all the documentation you need to start using Meilisearch's Javascript SDK.
+
+For general information on how to use Meilisearch, refer to our [main documentation website](https://docs.meilisearch.com/learn/) and the [Meilisearch API Reference](https://docs.meilisearch.com/reference/api/).
 
 ## 🔧 Installation
 
-We only guarantee that the package works with `node` >= 12 and `node` <= 16.
+We recommend installing `meilisearch-js` in your project with your package manager of choice.
 
-With `npm`:
+If you use `npm`:
 
 ```sh
 npm install meilisearch
 ```
 
-With `yarn`:
+If you prefer `yarn`:
 
 ```sh
 yarn add meilisearch
 ```
 
-### 🏃‍♀️ Run Meilisearch <!-- omit in toc -->
+`meilisearch-js` officially supports `node` versions >= 12 and <= 16.
+
+Instead of using a package manager, you may also import the library directly into your HTML via a CDN.
```

Comment:
```html
<script src="https://cdn.jsdelivr.net/npm/meilisearch@0.25.1/dist/bundles/meilisearch.umd.min.js"></script>
```

Should we add this?

Reply:
I like that idea. Do you think adding the CDN link here makes the import section down below redundant?